packages/middleware-endpoint/src/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from "./adaptors/getEndpointFromConfig";
+export * from "./adaptors/toEndpointV1";
```

Comment:
Create `index.ts` in adaptors folder to re-export the adaptors?
```suggestion
export * from "./adaptors";
```